pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Operations Features
+
+- Operations describe the behaviors and capabilities of the messaging channels described in the document.
+
+- In a messaging channel, an operation represents a particular action or interaction that can be performed. 
+
+- The purpose of these operations is to provide a standardized means for describing the process of publishing, subscribing to, requesting, or replying to messages within the messaging system.
+
+## Steps to Add Operations
+
+For adding operations to an AsyncAPI document, we need to define them within the channels section of the document. The step-by-step guide on how to add operations to an AsyncAPI document is as follows - 
+
+- Open your AsyncAPI document in a text editor or an AsyncAPI editor tool.
+
+- Locate the `channels` section in your AsyncAPI document. This section defines the messaging channels and their associated operations.
+
+- Within the `channels` section, add a new channel using the desired channel name. Channels are represented as properties within the `channels` object, and their names are usually formatted as paths or topics, depending on the messaging protocol you are using (e.g. `/users/{userId}/notifications` or `user.notifications`).
+
+- Inside the channel definition, specify the desired operations by creating sub-properties for each operation. Common operations include `publish`, `subscribe`, `publishSubscribe`, `requestReply`, and request.
+
+- For each operation, define its details using the appropriate keywords and properties. These details include the operation type, payload schema, headers, bindings, and other relevant information.
+
+```mermaid
```

Comment:
Would be nice to have a sentence introducing the diagram and what it illustrates?

Reply:
added a heading to the mermaid diagram describing what it is actually showing